operator/src/test/java/oracle/kubernetes/operator/helpers/CallBuilderTest.java

```diff
@@ -10,7 +10,7 @@
 import java.util.function.Consumer;
 
 import com.google.gson.GsonBuilder;
-import com.meterware.pseudoserver.HttpUserAgentTest;
+import com.meterware.pseudoserver.PseudoServer;
```

Comment:
where do we find out more about PseudoServer?

Reply:
It's part of HttpUnit. I hadn't remembered that I had put it in its own package hierarchy rather than <com.meterware.httpunit.pseudoserver>.